ui/src/me/components/account/__snapshots__/Settings.test.tsx.snap

```diff
@@ -50,74 +50,77 @@ exports[`Account rendering renders! 1`] = `
                   }
                 >
                   <PanelHeader
+                    alignItems="center"
```

Comment:
Whats this file?

Reply:
They are kinda worthless tests IMO, but they break if the DOM changes in any way for their associated components. You end up updating them all the time